### PR TITLE
scripts/tokens/Style Dictionary -> Old Token Format

### PR DESCRIPTION
**Pull request by chrisadubois**
# Guide

**This "Help" section can be deleted before submitting this pull request.**

*Update the name of this pull request to reflect the following shape:*

```
{type}/{scope?}/{message}
```

* **type** - A [conventional commit type](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional#type-enum) **REQUIRED**
* **scope** - The kabab-case scope of the changes in this request
* **message** - A short, kebab-case statement describing the changes **REQUIRED**

*Provide a general summary of the scope of the changes in this pull request.*

the core code format is done to read the "new" token format from style dictionary and figma tokens plugin output to the old tokens format so that we can fully move off to the abstract.

* [Readme](https://github.com/momentum-design/tokens/blob/master/README.md)
* [Getting Started Guide](https://github.com/momentum-design/tokens/blob/master/GETTING_STARTED.md)
* [Contributing Guide](https://github.com/momentum-design/tokens/blob/master/CONTRIBUTING.md)

# Description

*The full description of the changes made in this request.*

# Links

*Links to relevent resources.*

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,6 +52,7 @@
     "**/*": "npx prettier --write --ignore-unknown"
   },
   "dependencies": {
-    "momentum-abstract": "^1.0.28"
+    "momentum-abstract": "^1.0.28",
+    "@momentum-design/tokens": "file:/Users/chdubois/Projects/momentum-design/packages/@momentum-design/tokens"
```
Comment (chrisadubois) [inline package.json:56]: this will be removed

File: scripts/commands/update/update.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 const nextTokens = {
-  core: require("momentum-abstract/color/core.json"),
+  core: require("@momentum-design/tokens/dist/webex/json/core.json"),
```
Comment (chrisadubois) [inline scripts/commands/update/update.js:2]: as will this

File: scripts/commands/update/update.js [hunk 3]
```diff
@@ -19,22 +20,41 @@ const { ColorToken, GradientToken, SolidToken } = require("../../models");
 const update = () => {
   // Next tokens are mapped onto legacy tokens. These must be normalized to
   // match the existing local tokens.
+  // const next = {
```
Comment (chrisadubois) [inline scripts/commands/update/update.js:23]: @InteractiveTimmy -- I was wondering how you intended to handle all 3 formats -- standard, automated, and design tokens in the same merge flow, or if the next should just be the style dictionary format

File: yarn.lock [hunk 4]
```diff
@@ -567,6 +567,9 @@
     "@jridgewell/resolve-uri" "3.1.0"
     "@jridgewell/sourcemap-codec" "1.4.14"
 
+"@momentum-design/tokens@file:../momentum-design/packages/@momentum-design/tokens":
```
Comment (chrisadubois) [inline yarn.lock:570]: this will be removed

**Review by chrisadubois — COMMENTED** [review]
I will add unit tests -- wanted to get reviews on functionality, will not merge without UTs

File: scripts/models/token/token.constants.js [hunk 5]
```diff
@@ -25,6 +25,7 @@ const SERIAL_SPECIFICATIONS = {
 const TOKEN_FORMATS = {
   STANDARD: "standard",
   AUTOMATED: "automated",
+  DESIGN: "design",
```
Comment (cscheffauer) [inline scripts/models/token/token.constants.js:28]: can we somewhere document the different token formats? for me its not 100% clear whats the exact difference between those 3. thanks

File: scripts/commands/update/update.js [hunk 6]
```diff
@@ -16,27 +14,54 @@ const { ColorToken, GradientToken, SolidToken } = require("../../models");
...
+        format: ColorToken.CONSTANTS.TOKEN_FORMATS.AUTOMATED,
+        data: nextTokens.abstract["decorative color"],
+      }).normalize(),
+      gradation: new GradientToken({
+        format: ColorToken.CONSTANTS.TOKEN_FORMATS.AUTOMATED,
+        data: { gradation: nextTokens.abstract["gradation color"] }, // automated gradation token is missing the `gradation` key.
+      }).normalize(),
+      solids: new SolidToken({
+        format: ColorToken.CONSTANTS.TOKEN_FORMATS.AUTOMATED,
+        data: nextTokens.abstract["mobile solid background"],
```
Comment (cscheffauer) [inline scripts/commands/update/update.js:39]: not quite understanding why this is "mobile solid background" - why is this named / scoped "mobile"?

**Review by cscheffauer — APPROVED** [review]

**Pull request merged**